samples/ControlCatalog/SideBar.xaml

```diff
@@ -1,6 +1,14 @@
 <Styles xmlns="https://github.com/avaloniaui"
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
         x:Class="ControlCatalog.SideBar">
+    <Design.PreviewWith>
```

Comment:
I think it's not ignored in non-design mode, we might want to alter CompatibleXamlReader later or just wait for compiler to be ready.

Reply:
Yeah, I'd like to somehow make the XAML parser ignore `Design` attached properties. For the moment though, are you saying we should remove this?

Reply:
Since it's our control catalog, it shouldn't affect anything, so we can keep it.